assets/airports/lszh.json

```diff
@@ -0,0 +1,1812 @@
+            "rwy": {
+                "LSZH16": [["^_RWY16KLO01DME", "S210-"], ["_ZUE232007", "S210-"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        },
+        "ZUE2V": {
+            "icao": "ZUE2V",
+            "name": "Zurich East Two Victor",
+            "rwy": {
+                "LSZH28": [["^_RWY28KLO03DME", "S210-"], ["_KLO253005", "S210-"], ["^ZH552", "S210-"], "_KLO232005", "KLO"]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        },
+        "ZUE3L": {
+            "icao": "ZUE3L",
+            "name": "Zurich East Three Lima",
+            "rwy": {
+                "LSZH32": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+|S210-"], "_TRA190011", ["^_TRA190012.5", "A45+"], "_KLO269005.5", ["KLO", "A50+"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        },
+        "ZUE3F": {
+            "icao": "ZUE3F",
+            "name": "Zurich East Three Foxtrott",
+            "rwy": {
+                "LSZH34": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+|S210-"], "_TRA190011", ["^_TRA190012.5", "A45+"], "_KLO269005.5", ["KLO", "A50+"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "ZUE": [["^ZUE", "A60+"]]
+            },
+            "draw": [
+                ["_ZUE232007", "ZUE*"]
+            ]
+        }
+    },
+    "stars": {
+        "BERS1G": {
+            "icao": "BERS1G",
+            "name": "Bersu One Golf",
+            "entryPoints": {
+                "BERSU": ["BERSU"]
+            },
+            "body": ["TADOB", ["ERMUS", "A80+"], ["@GIPOL", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": [],
+                "LSZH32": [],
+                "LSZH34": []
+            },
+            "draw": [
+                ["BERSU", "TADOB", "ERMUS", "GIPOL"]
+            ]
+        },
+        "BLM2G": {
+            "icao": "BLM2G",
+            "name": "Basle-Mulhouse Two Golf",
+            "entryPoints": {
+                "BLM": [["BLM", "A200-"]]
+            },
+            "body": [["ZH677", "A120+"], ["@GIPOL", "A70+"]],
+            "rwy": {
+                "LSZH10": [],
+                "LSZH14": [],
+                "LSZH16": [],
+                "LSZH28": [],
+                "LSZH32": [],
+                "LSZH34": []
+            },
+            "draw": [
+                ["BLM", "ZH677", "GIPOL"]
+            ]
+        },
+        "DOPI1G": {
+            "icao": "DOPI1G",

```

Comment:
DOPIL2G

Reply:
I'm sure it's DOPIL1G, that's what jeppesen + vatsim charts say

Reply:
yup, DOPIL1G, sorry!